tests/behat/features/api/mediapackage/mediafile_search_get.feature

```diff
@@ -147,7 +147,7 @@ Feature: Get data from the media library in json format
     And I have a parameter "query_string" with value "Cat"
     And I have a parameter "flavor" with value "pocketcode"
     And I request "GET" "/api/media/file/search"
-    Then the response status code should be "200"
+    Then The response status code should be "200"
```

Comment:
"the" would be the best practice not "The"

The sentence starts with the keyword (Then, When, And, ...) 
so the first word of the step definition is not the first word of the sentence.

Therefore it should be in lower case, with exceptions as we know them from the english grammar.

https://docs.behat.org/en/latest/user_guide/gherkin.html